specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/virtualMachineScaleSet.json

```diff
@@ -1755,6 +1755,66 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmScaleSetName}/approverollingupgrade": {
```

Comment:
>subscriptions [](http://example.com/codeflow?start=6&length=13)

Address failures here - [https://github.com/Azure/azure-rest-api-specs/pull/25543/checks?check_run_id=17792796307](https://github.com/Azure/azure-rest-api-specs/pull/25543/checks?check_run_id=17792796307)

Reply:
I addressed the POST response code error and the 202 location header error. Error content schema can't really be addressed in this PR, seems that every single API in this file uses that v1, so I will skip addressing that.

Reply:
Please consider upgrade the common types when release this feature branch.